File: kyu6/build_tower.js
function towerBuilder(nFloors) {
    var pyr = [];
    for (var i = 0; i < nFloors; i++){pyr.push("*".repeat(i*2) + "*")}
    const def = pyr[pyr.length-1].length;
    for (var i = 0; i < nFloors; i++)
    {
    pyr[i] = " ".repeat((def-pyr[i].length)/2) + pyr[i] + " ".repeat((def-pyr[i].length)/2)
    }
    return pyr;
  }

File: kyu6/build_tower.py
def tower_builder(n_floors):
    max_tower_width = 2 * n_floors - 1
    return [floor_builder(max_tower_width, current_floor) for current_floor in range(n_floors)]

def floor_builder(max_tower_width, current_floor):
    # -(-bar//foo) rounds up instead of down
    spacing = " " * (-(-max_tower_width // 2) - 1 - current_floor)
    asterisks = "*" * (1 + current_floor * 2)
    return "{0}{1}{0}".format(spacing, asterisks)
